src/libraries/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.cs

```diff
@@ -84,10 +163,28 @@ public sealed partial class ExtensionDataObject
     {
         internal ExtensionDataObject() { }
     }
+    public sealed partial class GenericInfo
+    {
+        internal GenericInfo() { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public System.Xml.XmlQualifiedName GetExpandedStableName() { throw null; }
+        public System.Collections.Generic.IList<GenericInfo>? Parameters { get { throw null; } }
+        public System.Xml.XmlQualifiedName StableName { get { throw null; } }
+    }
     public partial interface IExtensibleDataObject
     {
         System.Runtime.Serialization.ExtensionDataObject? ExtensionData { get; set; }
     }
+    public sealed partial class XmlDataContract : DataContract
+    {
+        public bool IsAnonymous { get { throw null; } }
+        public bool IsTopLevelElementNullable { get { throw null; } }
+        public bool IsTypeDefinedOnImport { get { throw null; } set { throw null; } }
+        public void SetIsValueType(bool isValueType) { throw null; }
+        internal XmlDataContract(Type type) : base(new XmlDataContractCriticalHelper()) { }
+        public System.Xml.Schema.XmlSchemaType? XsdType { get { throw null; } }
```

Comment:
You don't need to add the internal type XmlDataContractCriticalHelper to the reference assembly as the base constructor just takes an XmlDataContractCriticialHelper. You can replace this with `base(default)` and remove XmlDataContractCriticialHelper from this file.